### PR TITLE
Add tag for resource groups

### PR DESCRIPTION
**Pull request by vnguyen7**
Add tags for resource group related to #7109 in order to fix from PR #7774 of joestump. (one last test )

### CODE REVIEW

File: aws/resource_aws_resourcegroups_group.go [hunk 1]
```diff
@@ -176,3 +216,45 @@ func resourceAwsResourceGroupsGroupDelete(d *schema.ResourceData, meta interface
 
 	return nil
 }
+
+func tagsToMapRG(ts map[string]interface{}) map[string]string {
```
Comment (ewbankkit) [inline aws/resource_aws_resourcegroups_group.go]: I think you can reuse the functionality in [`tagsGeneric.go`](https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/tagsGeneric.go) here.

Comment (vnguyen7) [reply aws/resource_aws_resourcegroups_group.go]: Many thanks !!!! Good to know since I'm struggle with the Tag function !!

**Comment by vnguyen7** [comment]
@bflad  Hi apologize about that. I just edited in order for #7774's credit.

**Comment by DrFaust92** [comment]
Hey @vnguyen7, i have a working PR (#10640) for this with changes including the keyvaluetags package, would you like me to merge these to this PR or close this one?

**Comment by bflad** [comment]
Hi @vnguyen7 👋 Thank you for working on this and sorry it was previously lingering.

As @DrFaust92 is mentioning above, there have been some consistency changes and simplifications to how to implement resource tagging into the Terraform AWS Provider codebase, which can be found in the [Contributing Guide](https://github.com/terraform-providers/terraform-provider-aws/blob/master/.github/CONTRIBUTING.md#adding-resource-tagging-support) using a new internal `keyvaluetags` package.

Since we didn't see a response to the above and we would prefer to implement the newer methodology, we are going to opt to close this. Thanks again for your contribution here and let us know if you have any questions.

**Pull request closed without merging**